core/org.wso2.carbon.core.services/src/main/java/org/wso2/carbon/core/services/authentication/AuthenticationAdmin.java

```diff
@@ -238,12 +240,16 @@ public void logout() throws AuthenticationException {
             if (delegatedBy == null && loggedInUser != null) {
                 String logMsg = "'" + loggedInUser + "@" + tenantDomain + " [" + tenantId + "]' logged out at " + date.format(currentTime);
                 log.info(logMsg);
-                audit.info(logMsg);
+                if (!Boolean.parseBoolean(System.getProperty(DISABLE_LEGACY_LOGS))) {

```

Comment:
shall we have a util method to get the value of Boolean.parseBoolean(System.getProperty(DISABLE_LEGACY_LOGS)?

Reply:
Fixed in b8f5176 and bfb6efb